python/cudf/cudf/tests/test_binops.py

```diff
@@ -563,12 +562,17 @@ def test_operator_func_between_series_logical(
         pdf_series_b, fill_value=fill_value
     )
 
+    if scalar_a in [None, np.nan] and scalar_b in [None, np.nan]:
```

Comment:
It's not immediately clear to me why we are doing the below `fillna` followed by `astype` - could we please include more details in the comments?

Reply:
how's the updated comment?

Reply:
So actually, `astype` isn't required. That's an artifact of me fixing the test. I'm removing it and updating the comment again.